src/scripts/main.js

```diff
@@ -1,3 +1,64 @@
 'use strict';
 
-// write your code here
+const rows = Array.from(document.querySelectorAll('tbody tr'));
+
+const dataObj = rows.map(row => {
```

Comment:
Instead of creating a data object, you can directly modify the table by appending the new cell.